nullaway/src/main/java/com/uber/nullaway/CodeAnnotationInfo.java

```diff
@@ -269,7 +270,14 @@ private boolean isAnnotatedTopLevelClass(Symbol.ClassSymbol classSymbol, Config
       // make sure it's not explicitly configured as unannotated
       return !shouldTreatAsUnannotated(classSymbol, config);
     }
-    return false;
+    // Check if it is NullMarked inside a Library Model when in JSpecify Mode
+    if (config.isJSpecifyMode()) {
+      // TODO: Need to replace with handler instance sent as param
```

Comment:
Yes, we should fix this :-)  It's a bit annoying to have to send everything through the single `Handler` object, and could be impacting efficiency.  But, we should do a re-design of that in a different PR.